minio/minio
# use statvfs(2) for disk.GetInfo on NetBSD

## Description

This change adds `stat_netbsd.go` to implement `disk.GetInfo` for NetBSD, similar to the Solaris implementation. Difference is in handling the file system type field.
Because `stat_fallback.go` has no more users it can be deleted.

## Motivation and Context

Minio server currently does not start on NetBSD because of missing disk.Getinfo implementation.

## How to test this PR?

Try to run server on NetBSD. Prior to the change it would fail as such:
```
ERROR Unable to initialize backend: Unable to write to the backend
      > Please ensure MinIO binary has write permissions for the backend
      HINT:
        The disk size is less than 900MiB threshold
```

## Types of changes

## Checklist:
